app/_hub/kong-inc/mtls-auth/index.md

```diff
@@ -53,7 +53,7 @@ params:
     - name: revocation_check_mode
       default: "`IGNORE_CA_ERROR`"
       description: |
-        Controls client certificate revocation check behavior. Valid values are `SKIP`, `IGNORE_CA_ERROR` or `STRICT`. If set to `SKIP`, no revocation check will be performed. If set to `IGNORE_CA_ERROR`, the plugin will respect the revocation status when either OCSP or CRL URL is set, and will not fail on network issues. If set to `STRICT`, the plugin will only treat the certificate as valid when it's able to verify the revocation status, and a missing OCSP or CRL URL in the certificate or a failure to connect to the server will result in a revoked status. If both OCSP and CRL URL are set, the plugin always checks OCSP first, and will only check CRL URL if it can't communicate with the OCSP server.
+        ***Known Issue: The default value `IGNORE_CA_ERROR` has a known issue in versions 1.5.0.0 and later. To workaround, manually set the value to `SKIP`***.  Controls client certificate revocation check behavior. Valid values are `SKIP`, `IGNORE_CA_ERROR` or `STRICT`. If set to `SKIP`, no revocation check will be performed. If set to `IGNORE_CA_ERROR`, the plugin will respect the revocation status when either OCSP or CRL URL is set, and will not fail on network issues. If set to `STRICT`, the plugin will only treat the certificate as valid when it's able to verify the revocation status, and a missing OCSP or CRL URL in the certificate or a failure to connect to the server will result in a revoked status. If both OCSP and CRL URL are set, the plugin always checks OCSP first, and will only check CRL URL if it can't communicate with the OCSP server.
```

Comment:
```suggestion
        ***Known Issue: The default value `IGNORE_CA_ERROR` has a known issue in versions 1.5.0.0 and later. To workaround, manually set the value to `SKIP`***.  Controls client certificate revocation check behavior. Valid values are `SKIP`, `IGNORE_CA_ERROR`, or `STRICT`. If set to `SKIP`, no revocation check will be performed. If set to `IGNORE_CA_ERROR`, the plugin will respect the revocation status when either OCSP or CRL URL is set, and will not fail on network issues. If set to `STRICT`, the plugin will only treat the certificate as valid when it's able to verify the revocation status, and a missing OCSP or CRL URL in the certificate or a failure to connect to the server will result in a revoked status. If both OCSP and CRL URL are set, the plugin always checks OCSP first, and will only check CRL URL if it can't communicate with the OCSP server.
```

Reply:
Due to the runon text on one line number, git can't diff this in a granular manner. This is what I suggested:

![image](https://user-images.githubusercontent.com/3756245/82506079-74ad3f80-9ac4-11ea-8c62-3ce5c295c7c3.png)